.github/workflows/scripts/ping-codeowners-prs.sh

```diff
@@ -58,7 +59,7 @@ main() {
         "https://api.github.com/repos/${REPO}/pulls/${PR}/requested_reviewers" \
         -d "{\"reviewers\":[${REVIEWERS}]}" \
         | jq ".message" \
-        || echo "Request failed to request review from code owners on #${PR}"
+        || echo "Request failed to request review from code owners"
 }
 
-main || echo "Failed to request review from code owners on PR #${PR}"
+main || echo "Failed to request review from code owners"
```

Comment:
Is this echo necessary? Wouldn't line 62 cover the case of the failed request and always return 0?

Reply:
It was meant to be a broad catch-all that would cover any cases that might appear in the function, since we want to guard against this workflow ever failing. I've added a note to the call to document this.